src/sonic-config-engine/tests/sample-dell-6100-t0-minigraph.xml

```diff
@@ -716,6 +716,11 @@
             <a:Reference i:nil="true"/>
             <a:Value>10.0.0.16</a:Value>
           </a:DeviceProperty>
+          <a:DeviceProperty>
+          <a:Name>SonicQosProfile</a:Name>
+          <a:Reference i:nil="true"/>
+          <a:Value>Rdma-centric</a:Value>
```

Comment:
RDMA-CENTRIC?

Reply:
not needed. L540 in parser takes care of converting it to uppercase